webapp/certified/views.py

```diff
@@ -424,7 +424,7 @@ def certified_component_details(component_id):
     machines = machines_by_id.values()
 
     return render_template(
-        "certified/component-details.html",
+        "certified/components/component-details.html",
```

Comment:
I moved all component-related templates under the `/certified/components` directory, so that we organize the files a bit more.